### DISCUSSION TITLE
Select data from table with condition having more than 1000 input parameters

### DISCUSSION THREAD

**Discussion by nagarajuakula** [dflib/dflib#400] [Q&A] [answered]
Hi,

I would like to retrieve data from a table or a dataframe where I have 1000 parameters in a condition. 

Ex: Select * from Employee where empId in (1,2,3,......, 2000);
In oracle, it will not allow to use more than 1000 params so we partition the input list by multiples of 1000 and run the same query multiple times. Do we need to handle it on our own with DataFrames as well or we have any features?

Appreciate any help.

Thanks
Nagaraju

**Comment by andrus** [comment, marked as answer]
Yes, you'll need to handle it on your own. There's nothing in DFLib that would split query parameter lists.

**Reply by nagarajuakula** [reply]
Sure. Thank you.
